File: round 1/GRYFFINDOR/problem1.java
public class problem1
{
public static void main(long n)
{
long i=0;
int p=0;
for(i=1;i<=n;i++)
{
if(n%i==0)
{
p++;
}
if(p==2)
System.out.println("Yes");
else
System.out.println("No");
}
}
}

File: round 1/AppyROCKS/problem1.cpp
#include<iostream>
#include<conio.h>

using namespace std;
int main()
{ 
 int n=0, c=0;
 cout<<"input no.";
 cin>>n; 
 for(int i=1;i<n;i++)
 {
   c=n%i;
   if(c!=0)
    cout<<"Yes";
   else
    cout<<"No";
  }
 getch();
}